corehq/apps/domain/models.py

```diff
@@ -438,6 +438,9 @@ class Domain(QuickCachedDocumentMixin, BlobMixin, Document, SnapshotMixin):
 
     requested_report_builder_subscription = StringListProperty()
 
+    # Update case search ES synchronously on web apps form submission
+    web_apps_sync_case_search = BooleanProperty(default=False)
```

Comment:
I think it would be better to put this on the `CaseSearchConfig` object instead

Reply:
`CaseSearchConfig` is a domain setting - that's the thing you edit [here](https://www.commcarehq.org/a/ethan-test/settings/project/case_search/)

Reply:
Totally forgot about that config. Updated in 5cfd726ea81